include/zephyr/bluetooth/classic/hfp_ag.h

```diff
@@ -11,6 +11,10 @@
 #ifndef ZEPHYR_INCLUDE_BLUETOOTH_HFP_AG_H_
 #define ZEPHYR_INCLUDE_BLUETOOTH_HFP_AG_H_
 
+#ifndef Z_API
```

Comment:
Z_API has been defined in include/zephyr/bluetooth/bluetooth.h, and it awalys true. I think here need to be remove